addons/crm/models/crm_lead.py

```diff
@@ -1231,13 +1231,16 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
-            ('alias_name', '!=', False),
-            ('alias_name', '!=', ''),
-            ('alias_model_id.model', '=', 'crm.lead'),
-            ('alias_parent_model_id.model', '=', 'crm.team'),
-            ('alias_force_thread_id', '=', False)
-        ], limit=1)
+        alias_domain = [
+            ('active', '=', True),
```

Comment:
Actually 'active' should already be implicit like always without `active_test=False` so not needed in the domain since it's a simple search now.